links.html

```diff
@@ -29,22 +29,21 @@
         </header>
 
         <main> 
-            <h1>  
-                <ul>
-                    <li> <a href="https://validator.w3.org/nu/" title="HTML Checker" target="_blank"> HTML Checker </a>  </li>
-                    <li> <a href="https://realfavicongenerator.net/" title="Favicon Generator" target="_blank"> Favicon Generator </a> </li>    
-                </ul>
-            </h1>
+            <h1>Useful links</h1>
+            <ul>
+                <li> <a href="https://validator.w3.org/nu/" title="HTML Checker" target="_blank"> HTML Checker </a>  </li>
+                <li> <a href="https://realfavicongenerator.net/" title="Favicon Generator" target="_blank"> Favicon Generator </a> </li>    
+            </ul>
         </main>
 
         <footer>  
             <hr>
                 <blockquote> 
                     "When you reach the end of your rope, tie a knot in it and hang on."
-                    <cite> <p> &mdash; <span class="author"> Franklin D. Roosevelt </span> </p> </cite>
+                    <cite> &mdash; <span class="author"> Franklin D. Roosevelt </span> </cite>
                 </blockquote>
 
-                <p> &copy; 2020 <a href="mailto:eric.dawson@student.saultcollege.ca?subject=Message from CSD120 Website"> Eric Dawson </a> </p>
+                <p> &copy; 2020 <a href="mailto:eric.dawson@student.saultcollege.ca?subject=Message%20from%20CSD120%20Website"> Eric Dawson </a> </p>
```

Comment:
Recall from the lecture slides that spaces in URLs should be encoded as the symbol `%20`